CMakeLists.txt

```diff
@@ -113,7 +113,9 @@ if(__BUILTIN_TRAP)
   set(HAVE_NORETURN_BUILTIN_TRAP 1)
 endif()
 
-find_package(LibRT)
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Android)
+  find_package(LibRT)
+endif()
```

Comment:
This is fine to do, but really unfortunate.  CMake already already can deal with librt properly on various targets, but we don't use CMake properly :-(.

Reply:
Yes, you could use `CMAKE_FIND_ROOT_PATH`, [like I do when cross-compiling the corelibs](https://github.com/apple/swift/pull/33724/files#diff-1a5a064af373f742294e3b312a5c8d0ab52f5aef78052490e6005165472cda48R2179), if there is an `librt.so` to find, but there is none on Android- it's just part of `libc.so`- so this is the right change for Android.